Scarab/Views/UninstallDependenciesConfirmationWindow.axaml.cs

```diff
@@ -0,0 +1,26 @@
+using Avalonia.Controls;
+using Avalonia.Markup.Xaml;
+using Scarab.ViewModels;
+
+namespace Scarab.Views
+{
+    public partial class UninstallDependenciesConfirmationWindow : Window
+    {
+        public UninstallDependenciesConfirmationWindow()
+        {
+            InitializeComponent();
+        }
+
+        public UninstallDependenciesConfirmationWindow(UninstallDependenciesViewModel context)
```

Comment:
Why does this needs its own ctor cant it be set like this
https://github.com/TheMulhima/Scarab/blob/29d288d1043b3f62cbe49c9c17b0399985c2ab0f/Scarab/Util/DisplayErrors.cs#L46-L49